lib/full-site-editing/block-templates.php

```diff
@@ -124,7 +124,7 @@ function _gutenberg_get_template_files( $template_type ) {
  * @return array Template.
  */
 function _gutenberg_add_template_part_area_info( $template_info ) {
-	if ( WP_Theme_JSON_Resolver::theme_has_support() ) {
+	if ( WP_Theme_JSON_Resolver::theme_has_support( $template_info['slug'] ) ) {
```

Comment:
The function doesn't accept any arguments.

Reply:
Thanks! Leftover from a previous iteration. I'll remove it.